<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>930</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>930</width>
    <height>411</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>思源黑体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>RAM Simulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>20</y>
      <width>151</width>
      <height>341</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>130</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>0</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignTop</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>1</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignTop</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignTop</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignTop</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignTop</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignTop</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignTop</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignTop</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>8</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignTop</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignTop</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>A</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignTop</set>
     </property>
    </row>
    <column>
     <property name="text">
      <string>操作</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>作业1 申请 130K</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignTop</set>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="0">
        <red>0</red>
        <green>190</green>
        <blue>190</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>作业2 申请 60K</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignTop</set>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>作业3 申请 100k</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignTop</set>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>作业2 释放 60K</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignTop</set>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>作业4 申请 200K</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignTop</set>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>作业3 释放 100K</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignTop</set>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>作业1 释放 130K</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignTop</set>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string>作业5 申请 140K</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignTop</set>
     </property>
    </item>
    <item row="8" column="0">
     <property name="text">
      <string>作业6 申请 60K</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignTop</set>
     </property>
    </item>
    <item row="9" column="0">
     <property name="text">
      <string>作业7 申请 50K</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignTop</set>
     </property>
    </item>
    <item row="10" column="0">
     <property name="text">
      <string>作业6 释放 60K</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignTop</set>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>711</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Mode</string>
    </property>
    <widget class="QRadioButton" name="radioButton_first">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>首次适应</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_best">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>最佳适应</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_worst">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>30</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>最差适应</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_reset">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>复位</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_next">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>320</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>下一句</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_graph">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>710</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_end">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>320</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>运行到底</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>100</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>100</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>640</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>347</x>
      <y>100</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>300</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>100</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>236</x>
      <y>100</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>200</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>458</x>
      <y>100</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>400</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>569</x>
      <y>100</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>500</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>100</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>600</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
